### DISCUSSION TITLE
为什么接口协议里没有DHCPv6协议

### DISCUSSION THREAD

**Discussion by Constantinefun** [coolsnowwolf/lede#9546] [Bugs / 问题反馈]
#电犀牛R68S##为什么接口协议里没有DHCPv6协议

**Comment by feilongplus** [comment]
顶一下，这个确实需要。请大神安排上，谢谢！

**Comment by EnnawYang** [comment]
你见或不见,它就在那里
![Snipaste_2022-06-13_00-07-10](https://user-images.githubusercontent.com/6705663/173242398-b46a6832-c534-4337-bc1c-8b2d89ef6194.png)

**Reply by zhang73331** [reply]
同样没有DHCPv6协议，你这个是那个版本？

**Comment by smomop** [comment]
#9557 看看能不能解决
